### DISCUSSION TITLE
Update Oracle Table from Mirth

### DISCUSSION THREAD

**Discussion by LanciaH** [nextgenhealthcare/connect#5278] [Q&A] [answered]
Hi All, Hope all is well. I have a channel that  uses a data reader to read values from an oracle table and sends data to a vendor.  That part is working.   I would like to update a table in oracle after the channel is finished processing.  I have see references to update statements through Mirth on the web.  My question is where in the channel do I put the update statement?

Thanks
Ralph

**Comment by pacmano1** [comment, marked as answer]
Unless you are sending messages in batch, you might want to do that right after you send each message to the vendor.    Can you better detail exactly how you are sending those messages?

There are many places to run the update, including on the db reader source tab, see the very bottom "Run Post Process SQL" part, another place is a response transformers (e.g. after a single HL7 message was posted and you checked the ACK or an http post was done and you checked the response), etc.

**Comment by LanciaH** [comment]
We send the vendor medication orders per inmates.  Source is DataReader and in the source we transform the message into JSON.

Thanks for the Run Post Process SQL tip, I'll look into it.

Thanks!
